src/Utility/NotificationManager.php

```diff
@@ -75,36 +76,47 @@ public static function instance($manager = null)
      */
     public function notify($data)
     {
-        $model = TableRegistry::get('Bakkerij/Notifier.Notifications');
```

Comment:
is it a fork from another plugin? If so, please fork it, if using another plugin as a base, please cite it in the readme to credit the original author